blas/blas32/blas32.go

```diff
@@ -6,8 +6,8 @@
 package blas32
 
 import (
-	"github.com/gonum/blas"
-	"github.com/gonum/blas/native"
+	"gonum.org/v1/gonum/blas"
+	"gonum.org/v1/gonum/blas/native"
```

Comment:
But that's not right, is it? The pure go implementations are staying in `gonum`. Either way, the first PR will be deleting all but the `cgo` packages so it doesn't really matter.